client/src/pages/Settings/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import { Button, Typography } from '@mui/material';
+import { Box } from '@mui/system';
+import DeleteForeverIcon from '@mui/icons-material/DeleteForever';
+import AvatarDisplay from '../../../components/AvatarDisplay/AvatarDisplay';
+import SettingHeader from '../../../components/SettingsHeader/SettingsHeader';
+import { User } from '../../../interface/User';
+import { makeStyles } from '@mui/styles';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+import { profilePhotoUpload, profilePhotoDelete } from '../../../helpers/APICalls/profilePhoto';
+import { useAuth } from '../../../context/useAuthContext';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'flex',
```

Comment:
There are some comments about in the Photo Upload UI PR, let's address those and then we can have another look at this PR :)